### PR TITLE
chore: Update Playwright to v1.37.1

### PR DESCRIPTION
**Pull request by mofojed**
- No longer need both the `playwright` and `@playwright/test` packages, just need `@playwright/test`

### CODE REVIEW

**Comment by mofojed** [comment]
Seems to be failing locally because the quick filter font doesn't seem to be styled correctly:
image

**Comment by mofojed** [comment]
Also failed the run: https://github.com/deephaven/web-client-ui/actions/runs/6004690989?pr=1467

File: packages/iris-grid/src/IrisGrid.tsx [hunk 1]
```diff
@@ -3194,6 +3194,7 @@ export class IrisGrid extends Component<IrisGridProps, IrisGridState> {
 
   handleRollupChange(rollupConfig: UIRollupConfig): void {
     log.info('Rollup change', rollupConfig);
+    log.info('Rollup change str', JSON.stringify(rollupConfig));
```
Comment (mattrunyon) [inline packages/iris-grid/src/IrisGrid.tsx:3197]: Did you still want the extra logging in this PR?

File: tests/docker-scripts/docker-compose.yml [hunk 2]
```diff
@@ -2,11 +2,13 @@ version: '3.8'
...
   dhc-server:
-    image: ghcr.io/deephaven/server:${DHC_VERSION:-edge}
+    image: d9faeab23d0f
     expose:
       - 10000
+    ports:
+      - 5005:5005
     environment:
-      - START_OPTS=-Xmx4g -DAuthHandlers=io.deephaven.auth.AnonymousAuthenticationHandler -Ddeephaven.console.type=python
+      - START_OPTS=-Xmx4g -DAuthHandlers=io.deephaven.auth.AnonymousAuthenticationHandler -Ddeephaven.console.type=python -agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=*:5005
```
Comment (mattrunyon) [inline tests/docker-scripts/docker-compose.yml:11]: Revert these changes

File: tests/table-operations.spec.ts [hunk 3]
```diff
@@ -107,6 +107,16 @@ async function artificialWait(page: Page, tableNumber = 0) {
...
+  page.on('websocket', ws => {
+    console.log(`WebSocket opened: ${ws.url()}>`);
+    // ws.on('framesent', event => console.log('framesent', event.payload));
+    // ws.on('framereceived', event =>
+    //   console.log('framereceived', event.payload)
+    // );
+    ws.on('socketerror', event => console.log('WebSocket error', event));
+    ws.on('close', event => console.log('WebSocket closed: ', event.isClosed));
+  });
+
```
Comment (mattrunyon) [inline tests/table-operations.spec.ts:119]: Remove?

File: packages/iris-grid/src/sidebar/RollupRows.tsx [hunk 4]
```diff
@@ -390,6 +390,8 @@ class RollupRows extends Component<RollupRowsProps, RollupRowsState> {
   sendChange(): void {
     const { onChange } = this.props;
     const { columns, showConstituents, showNonAggregatedColumns } = this.state;
+    log.info('MJB RollupRows.sendChange columns now', JSON.stringify(columns));
```
Comment (mattrunyon) [inline packages/iris-grid/src/sidebar/RollupRows.tsx:393]: Remove?

**Review by mattrunyon — CHANGES_REQUESTED** [review]

File: packages/iris-grid/src/IrisGrid.tsx (hunk 1 above)
Comment (mofojed) [reply packages/iris-grid/src/IrisGrid.tsx:3197]: Nope. D'oh

**Review by mattrunyon — APPROVED** [review]
